jiracmd/transition.go

```diff
@@ -165,6 +165,25 @@ func CmdTransition(o *oreo.Client, globals *jiracli.GlobalOptions, opts *Transit
 				return err
 			}
 		}
+
+		// if issueUpdate contains fields lets see if we can map them
+		// to their ids
+		if len(issueUpdate.Fields) > 0 {
+			fields, err := jira.GetFields(o, globals.Endpoint.Value)
+			if err != nil {
+				return err
+			}
+			for k, v := range issueUpdate.Fields {
+				for _, f := range fields {
+					if f.Name == k {
+						// re-map to field.id
+						issueUpdate.Fields[f.ID] = v
+						delete(issueUpdate.Fields, k)
```

Comment:
Probably need a `break` after this to quit the inner loop on match.

Reply:
oh good catch.